### ISSUE TITLE
about the default reid-model "lmbn_n_*.pt"

### ISSUE THREAD

**Issue by Hello-Water** [mikel-brostrom/yolov8_tracking#880]
### Search before asking

- [X] I have searched the Yolov8 Tracking [issues](https://github.com/mikel-brostrom/yolov8_tracking/issues) and found no similar bug report.

### Question

![image](https://github.com/mikel-brostrom/yolov8_tracking/assets/30867133/92840193-a662-4b52-a6df-324961a1622d)

When I use the reid model of the `lmbn_n_*` series, the code does not have any feedback, only an empty `run/track/exp` file is generated? But other models(osnet_*, resnet_*, mobilenetv2_*) work fine.

**Comment by mikel-brostrom** [comment]
Maybe you forgot the `--save` flag? Never experienced any problems with this. I just tried 

```
(YDSO) ➜  yolov8_tracking git:(master) ✗ python3 track.py --source 0 --show --imgs 640 --hide-label --reid-model lmbn_n_duke.pt --save
/home/mikel.brostrom/yolov8_tracking
track: yolo_model=/home/mikel.brostrom/yolov8_tracking/weights/yolov8n.pt, reid_model=lmbn_n_duke.pt, tracking_method=deepocsort, source=0, imgsz=[640], conf=0.5, device=, show=True, save=True, classes=None, project=/home/mikel.brostrom/yolov8_tracking/runs/track, name=exp, exists_ok=False, half=False, vid_stride=1, hide_label=True, hide_conf=False
/home/mikel.brostrom/yolov8_tracking/weights/yolov8n.pt
1/1: 0... Success ✅ (inf frames of shape 640x480 at 30.00 FPS)

Successfully loaded imagenet pretrained weights from "/home/mikel.brostrom/.cache/torch/checkpoints/osnet_x1_0_imagenet.pth"
0: 480x640 1 person, 867.8ms
0: 480x640 1 person, 10.7ms
0: 480x640 1 person, 10.5ms
...
Speed: 1.3ms preprocess, 19.4ms inference, 1.9ms postprocess, 85.1ms tracking per image at shape (1, 3, 640, 640)
Results saved to /home/mikel.brostrom/yolov8_tracking/runs/track/exp44
96 labels saved to /home/mikel.brostrom/yolov8_tracking/runs/track/exp44/labels
```

Everything gets saved correctly: video, MOT results, and per frame results :smile:

**Comment by Hello-Water** [comment]
> Maybe you forgot the --save flag?
I have added the save flag with `--save-vid` for the former version.

When I update the code to the latest version and use the commands you mentioned above (just change the `--source` with my loal video), the result is the same as I encountered before. I just tried 
```python
 python3 track.py --yolo-model weights/yolov8n.pt --source video/person.mp4 --show --imgs 640 --hide-label --save --reid-model weights/lmbn_n_duke.pt
``` 

What confuses me mostly is that only the `lmbn_n_*`  models don't work properly, while others(resnet, osnet, mobilenet...) work fine.
